### DISCUSSION TITLE
Renovate and Micronaut

### DISCUSSION THREAD

**Discussion by jjathman** [renovatebot/renovate#19604] [Request Help] [answered]
Does anyone use Renovate to manage their micronaut version updates? I think the problem is how Micronaut defines the version in its own custom block and not as a regular dependency. Is there a different way to set this up?

example: https://micronaut-projects.github.io/micronaut-gradle-plugin/latest/#_minimal_build

**Comment by jaecktec** [comment]
did you have any success? I was playing around with the custom managers, sadly without success so far

**Comment by Churro** [comment, marked as answer]
PR #26519 adds native support for micronaut to renovate.
